Submitted By:            Igor Živković <contact@igor-zivkovic.from.hr>
Date:                    2014-01-19
Initial Package Version: 4.2
Upstream Status:         Not reported
Origin:                  Me
Description:             Fixes man page installation if missing texinfo.

diff -Naur bash-4.2.orig/doc/Makefile bash-4.2/doc/Makefile
--- bash-4.2.orig/doc/Makefile	2014-01-19 16:28:44.960298285 +0100
+++ bash-4.2/doc/Makefile	2014-01-19 16:29:45.763545448 +0100
@@ -222,19 +222,13 @@
 		$(SHELL) $(SUPPORT_SRCDIR)/mkinstalldirs $(DESTDIR)$(htmldir) ; \
 	fi
 	
-install: info installdirs bash.info
+install: installdirs
 	-$(INSTALL_DATA) $(srcdir)/bash.1 $(DESTDIR)$(man1dir)/bash${man1ext}
 	-$(INSTALL_DATA) $(srcdir)/bashbug.1 $(DESTDIR)$(man1dir)/bashbug${man1ext}
 # uncomment the next lines to install the builtins man page
 #	sed 's:bash\.1:man1/&:' $(srcdir)/builtins.1 > $${TMPDIR:-/var/tmp}/builtins.1
 #	-$(INSTALL_DATA) $${TMPDIR:-/var/tmp}/builtins.1 $(DESTDIR)$(man1dir)/bash_builtins${man1ext}
 #	-$(RM) $${TMPDIR:-/var/tmp}/builtins.1
-	-if test -f bash.info; then d=.; else d=$(srcdir); fi; \
-	  $(INSTALL_DATA) $$d/bash.info $(DESTDIR)$(infodir)/bash.info
-# run install-info if it is present to update the info directory
-	if $(SHELL) -c 'install-info --version' >/dev/null 2>&1; then \
-		install-info --dir-file=$(DESTDIR)$(infodir)/dir $(DESTDIR)$(infodir)/bash.info; \
-	else true; fi
 # if htmldir is set, install the html files into that directory
 	-if test -n "${htmldir}" ; then \
 		$(INSTALL_DATA) $(srcdir)/bash.html $(DESTDIR)$(htmldir) ; \
